testhelpers/util.go

```diff
@@ -54,13 +56,17 @@ func MustGetFilecoinBinary() string {
 
 // GetFilecoinBinary returns the path where the filecoin binary will be if it has been built
 func GetFilecoinBinary() (string, error) {
-	gopath, err := GetGoPath()
-	if err != nil {
-		return "", err
+	bin, provided := testflags.BinaryPath()
+	if !provided {
+		gopath, err := GetGoPath()
+		if err != nil {
+			return "", err
+		}
+
+		bin = filepath.Join(gopath, "/src/github.com/filecoin-project/go-filecoin/go-filecoin")
```

Comment:
I think we should delete the fallback to GOPATH, but instead expect "./go-filecoin". Yes, this requires the right CWD, but the `binary-path` option supports other cases.

Reply:
See: https://github.com/filecoin-project/go-filecoin/pull/3409